### PR TITLE
Fix mac build

### PR DESCRIPTION
**Pull request by kmilo9999**
App supports mac os build and compilation
Superbuild support for mac os

### CODE REVIEW

**Comment by kmilo9999** [comment]
@mcmcgrath13 @mirestrepo @eldu  A quick review of this PR is appreciate it. I want to move to test the C.I with github actions

File: libs/GLMLoader/GLMLoader.h [hunk 1]
```diff
@@ -12,7 +12,7 @@
 //  See license.txt for further information.
 //  
 //  BROWN UNIVERSITY DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE WHICH IS 
-//  PROVIDED �AS IS�, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+//  PROVIDED �AS IS�, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
```
Comment (mcmcgrath13) [inline libs/GLMLoader/GLMLoader.h:15]: what happened to the characters here? and the copyright in the next file

File: CMakeLists.txt [hunk 2]
```diff
@@ -166,12 +180,13 @@ set(LIBRARIES
...
 message(STATUS "cppfs_LIBRARY ${cppfs_LIBRARIES}")
 
-if (UNIX)
-	set(LIBRARIES
-		${LIBRARIES}
-		stdc++fs
-	)
-endif (UNIX)
+#if (UNIX)
+#   if(NOT APPLE)
```
Comment (mcmcgrath13) [inline CMakeLists.txt:184]: delete commented out code

File: src/loader/LoadDataAction.cpp [hunk 3]
```diff
@@ -209,18 +208,27 @@ Volume* LoadDataAction::run(bool convert)
...
   time_t posix_time;
+   std::cerr << "INIT Load Image 0"  << std::endl;
+   
   if (helper::ends_with_string(m_folder, "desc"))
   {
+     std::cerr << "INIT Load Image 1"  << std::endl;
+   
     FILE* pFile;
     pFile = fopen(m_folder.c_str(), "r");
+     std::cerr << "INIT Load Image 1.2"  << " pFile "<< m_folder.c_str()<<std::endl;
```
Comment (mirestrepo) [inline src/loader/LoadDataAction.cpp:219]: why are you sending this to the error file? maybe you want to be using a logging that's a bit more configurable? I wouldn't expect info type of output in the error stream...

File: src/render/Shader.cpp [hunk 4]
```diff
@@ -152,7 +174,7 @@ unsigned int Shader::compileShader()
     glDeleteShader(fragmentShader);
     return 0;
   }
-
+ //std::cout << "compileShader shader 6" << std::endl;
```
Comment (mirestrepo) [inline src/render/Shader.cpp:177]: remove commented out debug info.... also consider a logging library to turn on/off DEBUG level info

File: src/render/VolumeRaycastShader.cpp [hunk 5]
```diff
@@ -290,7 +288,9 @@ void VolumeRaycastShader::render(glm::mat4& MVP, glm::mat4& clipPlane, glm::vec3
 
 void VolumeRaycastShader::initGL()
 {
+ // std::cout << "initGL shader 1" << std::endl;
   bindProgram();
+ // std::cout << "initGL shader 2" << std::endl;
```
Comment (mirestrepo) [inline src/render/VolumeRaycastShader.cpp:293]: Same here... remove or use logging library

File: src/vrapp/VolumeVisualizationApp.cpp [hunk 6]
```diff
@@ -511,14 +513,14 @@ void VolumeVisualizationApp::onRenderGraphicsContext(const VRGraphicsState& rend
       std::cout << "Error initializing GLEW." << std::endl;
     }
 #endif        
-
+std::cout << "init vizapp " << std::endl;
```
Comment (mirestrepo) [inline src/vrapp/VolumeVisualizationApp.cpp:516]: consider logging library

**Review by mirestrepo — APPROVED** [review]
I would suggest using a logging library so you can easily change the verbosity of you logging without cluttering the code with commented out `cost` statements. But my recommendation is non-blocking so I approve.

File: CMakeLists.txt (hunk 2 above)
Comment (kmilo9999) [reply CMakeLists.txt]: Done

File: libs/GLMLoader/GLMLoader.h (hunk 1 above)
Comment (kmilo9999) [reply libs/GLMLoader/GLMLoader.h]: Maybe the character encoding replaced those characters when I edited them on mac. I changed them back to their original value.

File: src/render/Shader.cpp (hunk 4 above)
Comment (kmilo9999) [reply src/render/Shader.cpp]: I removed them.

File: src/vrapp/VolumeVisualizationApp.cpp (hunk 6 above)
Comment (kmilo9999) [reply src/vrapp/VolumeVisualizationApp.cpp:516]: this one might do the trick:

  https://github.com/gabime/spdlog

we can put it as an additional issue

File: src/render/VolumeRaycastShader.cpp (hunk 5 above)
Comment (kmilo9999) [reply src/render/VolumeRaycastShader.cpp]: I Removed them

**Pull request merged**